packages/node/test/client.test.ts

```diff
@@ -1,34 +1,33 @@
 import { ExperimentClient } from 'src/client';
 import { ExperimentUser } from 'src/types/user';
 
-const API_KEY = 'client-DvWljIjiiuqLbyjqdvBaLFfEBrAvGuA3';
+const API_KEY = 'server-qz35UwzJ5akieoAdIgzM4m9MIiOLXLoz';
```

Comment:
is it common practice to do an integration test in the sdk? I was assuming we would just mock out or hardcode expected responses and test the remaining eval logic?

Reply:
also, will be nice to not have a api key hardcoded in a public repo code like this

Reply:
Not necessarily common practice, no. It just makes testing a whole lot easier since I don't have to mock anything out. 

Eventually we want unit tests where the network is mocked, but that requires a bit more work.